stanford_profile_helper.module

```diff
@@ -268,11 +268,25 @@ function stanford_profile_helper_form_node_form_alter(&$form, FormStateInterface
   $hours = (int) floor($scheduler_increment / 3600);
   $mins = (int) floor($scheduler_increment / 60 % 60);
   $scheduler_increment = $hours ? "$hours hour(s)" : "$mins minute(s)";
+  $example_start = new DateTime('today 8:00 AM');
```

Comment:
What do you think about using `\Drupal\Core\Datetime\DrupalDateTime` class instead?

Reply:
Well, it looks like that class is a wrapper around DateTimePlus, which itself is a wrapper around the php-native DateTime.  It's not immediately obvious to me what benefits we'd derive from the DrupalDateTime, since essentially we're just doing a little quick math for a help message, but I'm pretty confident it wouldn't hurt anything either.  Are there benefits I'm not thinking of?

Reply:
No clue honestly. I know it's best to use that when displaying a time based on timezones, but as you said, it's not really timezone based here. :shrug:

Reply:
Yeah, we're really just looking for a couple of relative times without regard for TZ or anything.  It's good to know that class exists if it abstracts some of that stuff out effectively.  In the past, I've used a PHP library called Carbon for simplifying date stuff, and it sounds like there's some overlap in functionality there.  https://github.com/briannesbitt/carbon